test/parallel/test-error-format-list.js

```diff
@@ -0,0 +1,18 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+const { strictEqual } = require('node:assert');
+const { formatList } = require('internal/errors');
+
+if (common.hasIntl) {
```

Comment:
We want to print a skip message I think.
```suggestion
if (!common.hasIntl) common.skip('missing Intl');

{
```